applications/orbovideosr/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
Can you please add the license in this file as well.

Reply:
Missing copyright header